R/brreg.R

```diff
@@ -25,15 +25,51 @@
 #' df <- get_entity(c(974760843, 971524960))
 #'
 get_entity <- function(entity, raw_response = FALSE) {
-  dl <- lapply(entity, function(x) get_entity_safe(x, raw_response))
-  if (raw_response) return(dl)
+  dl <- lapply(entity, function(x)
+    get_brreg_safe(x, type = 'enheter', raw_response))
+  if (raw_response) return(invisible(dl))
   do.call('rbind', dl)
 }
 
+#' Get roles
+#'
+#' Fetch information about entity roles from the
+#' _Central Coordinating Register for Legal Entities_
+#' (Enhetsregisteret) based on the organisation number.
+#'
+#' @details
+#' See the
+#' [API documentation](https://data.brreg.no/enhetsregisteret/api/docs/index.html)
+#' for further details (in Norwegian only).
+#'
+#' @param entity A Norwegian organisation number.
+#' @inheritParams get_entity
+#' @return list
+#' @export
+#' @examples
+#' # Get roles for a single entity
+#' res <- get_roles(974760843)
+#'
+#' # Get roles for a mulitiple enities
+#' res <- get_roles(c(974760843, 971524960))
+#'
+get_roles <- function(entity, raw_response = FALSE) {
+  dl <- lapply(entity, function(x)
+    get_brreg_safe(x, type = 'roller', raw_response))
+  if (raw_response) return(invisible(dl))
```

Comment:
Should the difference in return value between `raw_response = FALSE` and `raw_response = TRUE` be explained (same comment applies for `get_entity()`.

Reply:
Good point. I added this to the documentation under details, for all three exported Brreg-function, similar to what you did for NB. 

I do think however that it is a bit tedious to repeat this for every single exported function, so as the package grows maybe we could find a way to either more smoothly inherit this section in the documentation, or make it clear in an overall README or introduction vignette.

Reply:
Agreed. This is something we should look into, and that fits with a DRY coding style.